Allow configuration from env vars

hectorhuertas opened the pull request:
I'll add error checking and reporting in another PR

hectorhuertas committed:
Accept array of extractors

hectorhuertas committed:
Read elasticsearch custom template from env var

hectorhuertas committed:
Add documentation of the available environmental variables

george-angel commented on README.md:
```diff
@@ -1,2 +1,50 @@
+
+## Configuration through environmental variables
+* `ADMIN_PASSWORD`: graylog api admin password
+* `EXTRACTORS`: space separated list of fields to be json parsed and ingested as indexed fields. Eg: `"kubernetes message log message_log"`
```
How do we specify the prefix for the extracted fields?

hectorhuertas replied:
We don't. The prefix will be `extractor_name`+`_`. Simple and easy to predict

george-angel replied:
As discussed, lets use `.` as extractor - field separator.

george-angel committed:
sys: name the executable (#9)

from: https://google.github.io/styleguide/shell.xml

> It is not necessary to know what language a program is written in when
> executing it and shell doesn't require an extension so we prefer not to
> use one for executables.

Also rename to something meaningful so listing procs on the box allows
you to discern them

hectorhuertas committed:
Use dot instead of underscore as separator

hectorhuertas committed:
Clarify extractors documentation

hectorhuertas committed:
Fix Dockerfile typo
